open-api/rest-catalog-open-api.yaml

```diff
@@ -1463,6 +1673,14 @@ components:
         type: string
       example: "sales"
 
+    id:
```

Comment:
I think we need to use a more complete name like `plan-id` here.  It's very possible this will collide (and it's not very descriptive as just `id`)

Reply:
makes sense will fix.